stglib/exo.py

```diff
@@ -457,7 +461,7 @@ def ds_add_attrs(ds):
     if "OST_62" in ds:
         ds["OST_62"].attrs.update(
             {
-                "units": "percent",
+                "units": "100",
```

Comment:
I am not sure about this. "percent" and "%" are both units in UDUNITS. See https://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#units

Reply:
Changed units back to percent and committed.